doc/en/components/grids/_shared/cell-editing.md

```diff
@@ -201,6 +202,97 @@ This code is used in the sample below which implements an [SelectComponent](../s
 
 <!-- end: Angular -->
 
+<!-- Blazor -->
+
+If you want to provide a custom template which will be applied to a cell, you can pass such template either to the cell itself, or to its header. 
+
+```Razor
+
+<IgbColumn
+            Field="AnnualChange"
+            Header="Annual Change"
+            DataType="GridColumnDataType.Number"
+            BodyTemplateScript="WebGridCurrencyCellTemplate"
+            HeaderTemplateScript="WebGridPinHeaderTemplate"
+            Name="column9"
+            @ref="column9">
+            </IgbColumn>
+
+
+```
+
+and pass the template:
+
+```javascript
+
+igRegisterScript("WebGridCurrencyCellTemplate", (ctx) => {
+    var html = window.igTemplating.html;
+    if (ctx.cell.value > 0) {
+        return html`<div>
+        <igc-badge variant="success"><span>▲</span></igc-badge>
+        <span style='color:green;'>${ctx.cell.value}</span>
+        </div>`;
+    } else {
+        return html`<div>
+        <igc-badge variant="danger"><span>▼</span></igc-badge>
+        <span style='color:red;'>${ctx.cell.value}</span>
+        </div>`;
+    }
+}, false);
+
+```
+Working sample of the above can be found here for further referencee: 
+
+`sample="/{ComponentSample}/column-moving-options", height="650", alt="{Platform} {ComponentTitle} Cell Editing Template Sample"`
+
+<!-- end: Blazor -->
+
+<!-- WebComponents -->
+
+If you want to provide a custom template which will be applied to a cell, you can pass such template either to the cell itself, or to its header. First create the column as you usually would:
```

Comment:
Same as for the blazor section. It does not show how to set cell editing template.
Should show an example on how to set `inlineEditorTemplate` to a column.